src/Components/WeatherCard/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useContext } from "react";
```

Comment:
there are import from react here and then below as well why?

Reply:
I will remove them